src/controllers/controller.cpp

```diff
@@ -141,7 +141,7 @@ void Controller::receive(const QByteArray data, mixxx::Duration timestamp) {
         }
         function.append(".incomingData");
         QJSValue incomingData = m_pEngine->wrapFunctionCode(function, 2);
```

Comment:
This variable name is confusing. Please rename it to `incomingDataFunction`. It took me a few minutes to verify that the call to `m_pEngine->executeFunction` below wasn't mistaking data for a function.